packages/compass-collection-stats/src/components/collection-stats-item/collection-stats-item.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { css } from '@mongodb-js/compass-components';
+import { uiColors } from '@leafygreen-ui/palette';
+
+const collectionStatsItemStyles = css({
+  marginRight: '12px',
+});
+
+const PrimaryLabelStyles = css({
+  textTransform: 'uppercase',
+  fontSize: '10px',
+  fontWeight: 'bold',
+  display: 'inline-block',
+  marginRight: ' 5px',
+  color: uiColors.gray.dark3,
+});
+
+const LabelStyles = css({
+  textTransform: 'uppercase',
+  fontSize: '10px',
+  fontWeight: 'bold',
+  color: uiColors.gray.base,
+  lineHeight: '12px',
+});
+
+const PrimaryValueStyles = css({
```

Comment:
Do any of the leafygreen `Typography` items fit the use case here?
https://www.mongodb.design/component/typography/example/
If so we'll be able to reduce the custom variable declaration here.

Reply:
Yeah, I want to do it in a separate pr. This one is mostly about the transition to typescript. Is it ok?

Reply:
Makes sense, thanks for clarifying!